thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/controller.js

```diff
@@ -302,6 +279,19 @@ export default Controller.extend({
     }
   ),
 
+  /**
+   * Find the active baseline option name
+   * @type {String}
+   */
+  baselineTitle: computed(
+    'baselineOptions',
+    function() {
+      const activeOpName = this.get('baselineOptions').filter(item => item.isActive).pop().name;
```

Comment:
is the last item in the array always guaranteed to be the active one?

Reply:
Its actually the first one (working because there is only one 'active' at a time). Changing it from .pop to [0].